kubetest2-gke/deployer/deployer.go

```diff
@@ -82,20 +85,23 @@ type deployer struct {
 	// doInit helps to make sure the initialization is performed only once
 	doInit sync.Once
 	// gke specific details
-	project           string
+	projects          []string
 	zone              string
 	region            string
-	cluster           string
+	clusters          []string
+	clustersLayout    map[string][]string
```

Comment:
nit: maybe rename to projectClusterLayout ?
also add a comment explaining the mapping.

Reply:
Done.